### PR TITLE
Update screenshots for Druid console doc

### PR DESCRIPTION
**Pull request by vtlim**
This PR updates screenshots in the Druid console doc and standardizes capitalization of "Druid console" in the docs.

This PR has:
- [x] been self-reviewed.

### CODE REVIEW

File: docs/operations/druid-console.md [hunk 1]
```diff
@@ -41,36 +39,50 @@ http://<ROUTER_IP>:<ROUTER_PORT>
...
 
 ![data-loader-2](../assets/web-console-03-data-loader-2.png)
 
 ## Datasources
 
-The datasources view shows all the currently enabled datasources.
-From this view, you can see the sizes and availability of the different datasources.
-You can edit the retention rules, configure automatic compaction, and drop data.
-Like any view that is powered by a DruidSQL query you can click `View SQL query for table` from the `...` menu to run the underlying SQL query directly.
+The datasources view shows all the datasources currently loaded on the cluster, as well as their sizes and availability.
```
Comment (ektravel) [inline docs/operations/druid-console.md:79]:
```suggestion
The **Datasources** view shows all the datasources currently loaded on the cluster, as well as their sizes and availability.
```

File: docs/operations/druid-console.md [hunk 2]
```diff
@@ -102,27 +117,30 @@ Click on the magnifying glass for any task to see more detail about it.
 
 ![tasks-status](../assets/web-console-09-task-status.png)
 
-## Servers
+## Services
 
-The servers tab lets you see the current status of the nodes making up your cluster.
+The services view lets you see the current status of the nodes making up your cluster.
```
Comment (ektravel) [inline docs/operations/druid-console.md:122]:
```suggestion
The **Services** view lets you see the current status of the nodes making up your cluster.
```

**Review by ektravel — APPROVED** [review]
I added some capitalization suggestions for various views in the console but those are all optional.

File: docs/operations/druid-console.md [hunk 3]
```diff
@@ -41,36 +39,50 @@ http://<ROUTER_IP>:<ROUTER_PORT>
 will show console users the files that the underlying user has permissions to. In general, avoid running Druid as 
 root user. Consider creating a dedicated user account for running Druid.
 
-Below is a description of the high-level features and functionality of the Druid Console
+This topic presents the high-level features and functionality of the Druid console.
 
 ## Home
 
-The home view provides a high level overview of the cluster.
+The home view provides a high-level overview of the cluster.
```
Comment (vtlim) [inline docs/operations/druid-console.md:46]:
```suggestion
The **Home** view provides a high-level overview of the cluster.
```

File: docs/operations/druid-console.md [hunk 4]
```diff
@@ -41,36 +39,50 @@ http://<ROUTER_IP>:<ROUTER_PORT>
...
 Each card is clickable and links to the appropriate view.
-The legacy menu allows you to go to the [legacy coordinator and overlord consoles](./management-uis.md#legacy-consoles) should you need them.
+
+The home view displays the following cards:
```
Comment (vtlim) [inline docs/operations/druid-console.md:49]:
```suggestion
The **Home** view displays the following cards:
```

File: docs/operations/druid-console.md [hunk 5]
```diff
@@ -41,36 +39,50 @@ http://<ROUTER_IP>:<ROUTER_PORT>
...
+
+* __Status__. Click this card for information on the Druid version and any extensions loaded on the cluster.
+* [Datasources](#datasources)
+* [Segments](#segments)
+* [Supervisors](#supervisors-and-tasks)
+* [Tasks](#supervisors-and-tasks)
+* [Services](#services)
+* [Lookups](#lookups)
+
+You can access the [data loader](#data-loader) and [lookups view](#lookups) from the top-level navigation of the home view.
```
Comment (vtlim) [inline docs/operations/druid-console.md:59]:
```suggestion
You can access the [data loader](#data-loader) and [lookups view](#lookups) from the top-level navigation of the **Home** view.
```

File: docs/operations/druid-console.md [hunk 6]
```diff
@@ -22,17 +22,15 @@ title: "Web console"
...
 
-The Druid Console is hosted by the [Router](../design/router.md) process.
-
-The following cluster settings must be enabled, as they are by default:
-
-- the Router's [management proxy](../design/router.md#enabling-the-management-proxy) must be enabled.
-- the Broker processes in the cluster must have [Druid SQL](../querying/sql.md) enabled.
-
-You can access the Druid console at:
+The following cluster settings must be enabled to use the Druid console. Note that these settings are enabled by default.
```
Comment (techdocsmith) [inline docs/operations/druid-console.md:28]:
```suggestion
Enable the following cluster settings o use the Druid console. Note that these settings are enabled by default.
```

File: docs/operations/druid-console.md [hunk 7]
```diff
@@ -41,36 +39,50 @@ http://<ROUTER_IP>:<ROUTER_PORT>
...
+* [Lookups](#lookups)
+
+You can access the [data loader](#data-loader) and [lookups view](#lookups) from the top-level navigation of the **Home** view.
 
 ![home-view](../assets/web-console-01-home-view.png "home view")
 
 ## Data loader
 
-The data loader view allows you to load data by building an ingestion spec with a step-by-step wizard.
+The data loader allows you to load data by building an ingestion spec with a step-by-step wizard.
```
Comment (techdocsmith) [inline docs/operations/druid-console.md:65]:
```suggestion
You can use the data loader build an ingestion spec with a step-by-step wizard.
```

File: docs/operations/druid-console.md [hunk 8]
```diff
@@ -102,27 +117,30 @@ Click on the magnifying glass for any task to see more detail about it.
...
+The **Query** view lets you issue [Druid SQL](../querying/sql.md) queries and display the results as a table.
+The view will attempt to infer your query and let you modify the query via contextual actions such as adding filters and changing the sort order when possible.
+
+From the ellipsis menu beside **Run**, you can view your query history, see the native query translation for a given Druid SQL query, and set the [query context](../querying/query-context.md).
 
 ![query-sql](../assets/web-console-11-query-sql.png)
 
 The query view can also issue queries in Druid's [native query format](../querying/querying.md), which is JSON over HTTP.
-To send a native Druid query, you must start your query with `{` and format it as JSON.
+To send a native Druid query, start your query with `{` and format it as a JSON object.
```
Comment (techdocsmith) [inline docs/operations/druid-console.md:137]: Don't all JSON objects begin with `{`?

**Review by techdocsmith — CHANGES_REQUESTED** [review]
Some of the captions could use improvement. No good way to comment on them here. We can chat about it.

File: docs/operations/druid-console.md (hunk 7 above)
Comment (vtlim) [reply docs/operations/druid-console.md:65]:
```suggestion
You can use the data loader to build an ingestion spec with a step-by-step wizard.
```

File: docs/operations/druid-console.md [hunk 9]
```diff
@@ -102,27 +117,30 @@ Click on the magnifying glass for any task to see more detail about it.
...
-The query view lets you issue [DruidSQL](../querying/sql.md) queries and display the results as a table.
-The view will attempt to infer your query and let you modify via contextual actions such as adding filters and changing the sort order when possible.
+The **Query** view lets you issue [Druid SQL](../querying/sql.md) queries and display the results as a table.
+The view will attempt to infer your query and let you modify the query via contextual actions such as adding filters and changing the sort order when possible.
+
+From the ellipsis menu beside **Run**, you can view your query history, see the native query translation for a given Druid SQL query, and set the [query context](../querying/query-context.md).
 
 ![query-sql](../assets/web-console-11-query-sql.png)
 
 The query view can also issue queries in Druid's [native query format](../querying/querying.md), which is JSON over HTTP.
```
Comment (vtlim) [inline docs/operations/druid-console.md:136]:
```suggestion
You can also use the query editor to issue queries in Druid's [native query format](../querying/querying.md), which is JSON over HTTP.
```

File: docs/operations/druid-console.md (hunk 8 above)
Comment (vtlim) [reply docs/operations/druid-console.md:137]: True. Rewording the previous sentence and removing the redundancy

Comment (vtlim) [reply docs/operations/druid-console.md:137]:
```suggestion
```

**Review by techdocsmith — APPROVED** [review]
LGTM

**Pull request merged**